app/modules/backup/continuous-replication/index.js

```diff
@@ -100,10 +101,11 @@ export default angular.module('backup.continuousReplication', [
       this.formData.tag = tag
       this.formData.selectedSr = selectedSr
       this.formData.scheduleId = schedule.id
+      this.formData.reportWhen = reportWhen
       this.scheduleApi.setCron(cronPattern)
     }
 
-    this.save = (id, vms, tag, sr, cron, enabled) => {
+    this.save = (id, vms, tag, sr, cron, enabled, reportWhen) => {
```

Comment:
It's quite clever to use a *virtual* param to pass data to the plugin but I think we should probably prefix it to avoid future name clashes, maybe with an underscore (`_`).